src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,148 @@
+CockroachDB is made available under the [CockroachDB Software License][csl].
+
+## Types of licenses
+
+The table of licenses below refers to options for {{ site.data.products.core }} deployments. All {{ site.data.products.cloud }} deployments automatically have a valid **Enterprise** license.
+
+Type | Description
+-------------|------------
+**Enterprise** <a name="enterprise"></a> | This paid license allows usage of all CockroachDB features with no restrictions under the terms specified in the [CockroachDB Software License][csl]. License must be renewed annually or as negotiated. Support levels available include [Enterprise][support] or [Essential][support].
+**Enterprise Free** <a name="enterprise-free"></a> | Same functionality as **Enterprise**, but free of charge for businesses with less than $10M in annual revenue, and telemetry is required except for ephemeral clusters (7 days or less). Clusters will be [throttled](#throttling) after 7 days without sending telemetry. License must be renewed annually. Support level available is Community (i.e., [Docs]({% link {{ page.version.version }}/index.md %}), [Forum][forum], [Slack][slack]).
```

Comment:
I feel like the "ephemeral" language here is confusing. I would delete ", and telemetry is required except for ephemeral clusters (7 days or less)".

Reply:
why do we mention the "7 days without sending telemetry" for the Enterprise Free license and not the Enterprise Trial license? I think these should be the same in terms of telemetry.

Reply:
> I feel like the "ephemeral" language here is confusing. I would delete ", and telemetry is required except for ephemeral clusters (7 days or less)".

ok will remove it

> why do we mention the "7 days without sending telemetry" for the Enterprise Free license and not the Enterprise Trial license? I think these should be the same in terms of telemetry.

no reason, human error.  Updated as shown below (removed ephemeral bit from Free, added the 7 days to Trial):

**Enterprise Free** <a name="enterprise-free"></a> | Same functionality as **Enterprise**, but free of charge for businesses with less than $10M in annual revenue. Clusters will be [throttled](#throttling) after 7 days without sending telemetry. License must be renewed annually. Support level available is Community (i.e., [Docs]({% link {{ page.version.version }}/index.md %}), [Forum][forum], [Slack][slack]).
**Enterprise Trial** <a name="enterprise-trial"></a> | A 30 day self-service trial license. Telemetry is required during the trial. Clusters will be [throttled](#throttling) after 7 days without sending telemetry. Telemetry can be disabled once the cluster is upgraded to a paid **Enterprise** license. Support level available during trials is Community (i.e., [Docs]({% link {{ page.version.version }}/index.md %}), [Forum][forum], [Slack][slack]).